communications/blogs/turing_joins_binder_federation.md

```diff
@@ -0,0 +1,102 @@
+# The Turing Institute invests in Open Infrastructure for Reproducible Research
+
+_This is an invited post from The Turing Way and The Alan Turing Institute, collaborators on the Binder project and new members of the Binder Federation._
+
+We are delighted to announce the fourth member of the Binder Federation: [The Alan Turing Institute](https://www.turing.ac.uk/)!
+Based in London UK, their infrastructure is hosted on [Microsoft Azure](https://azure.microsoft.com/en-gb/) cloud and is supported by [_The Turing Way_](https://github.com/alan-turing-institute/the-turing-way) project.
+
+## Reproducible and Transparent Science Communication
+
+Science moves forward by corroboration and the reproducibility crisis in science is [well known](https://www.nature.com/collections/prbfkwmwvz).
+When a paper is published, there is generally no way for an outsider to verify its results because the underlying data and analysis code are not available for scrutiny.
+This makes the science very difficult to build upon and results in fragile advances and lots of wasted time and money.
+As well as a cultural change in our approach to data science (such as the abolition of 'publish or perish' incentives, secrecy around data, and the drive for novelty), there is also a need for a practical change in the way science is communicated so that it is more transparent regarding the steps that were taken to produce the results.
```

Comment:
```suggestion
In addition to a cultural change in our approach to data science (such as the abolition of 'publish or perish' incentives, secrecy around data, and the drive for novelty), there is also a need for a practical change in the way science is communicated so that it is more transparent regarding the steps that were taken to produce the results.
```